code/game/objects/items/toys.dm

```diff
@@ -1489,3 +1491,34 @@
 	icon_state = "shell[rand(1,3)]"
 	color = pickweight(possible_colors)
 	setDir(pick(GLOB.cardinals))
+
+/obj/item/toy/brokenradio
+	name = "broken radio"
+	desc = "An old radio that produces nothing but static when turned on."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "broken_radio"
```

Comment:
you could also use the nuke op radio uplink sprite

Reply:
I think the Geiger counter Esq sprite works well as it keeps with the theme. On donut, there is already an "old radio" that uses the sprite

Reply:
Kilo uses the old radio too, in the bar and the atrium